packages/components/dialog/__tests__/dialog.spec.ts

```diff
@@ -6,25 +6,14 @@ import { Delete } from '@element-plus/icons-vue'
 import Dialog from '../src/dialog.vue'
 
 const AXIOM = 'Rem is the best girl'
-
-const _mount = ({ slots, ...rest }: Record<string, any>) => {
-  return mount(Dialog, {
-    slots: {
-      default: AXIOM,
-      ...slots,
-    },
-    ...rest,
-  })
-}
+const defaultSlots = { default: AXIOM }
```

Comment:
```suggestion
const defaultSlots = { default: () => AXIOM }
```

Reply:
Why, `AXIOM` is a text node

Reply:
But slots should be functions instead of plain objects

Reply:
Both are acceptable.

Reply:
Refer: https://vuejs.org/guide/extras/render-function.html#passing-slots

> Instead of an array, we need to pass either a slot function, or an object of slot functions

Reply:
This is rather a convention instead of a rule because of:

> Passing slots as functions allows them to be invoked lazily by the child component. This leads to the slot's dependencies being tracked by the child instead of the parent, which results in more accurate and efficient updates.

Reply:
I know that your code was not syntactically wrong but shouldn't we follow the convention?

Reply:
Updated.